### PR TITLE
Changed home page for Desktop and Mobile views

### PR DESCRIPTION
**Pull request by Pranav-Agarwal**
Changed the banner for the Desktop view and added a login box. Changed the mobile view to General Notices and added a seperate login page.

### CODE REVIEW

File: swd/frontend/src/Routes/login/Loginbox.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+  username: {
+    borderStyle: 'solid',
+    borderColor: '#074F57',
+    fontFamily: 'Montserrat',
+    color:'#074F57',
+    paddingLeft: 10,
+    borderWidth: 2,
+    fontSize: 17,
+    marginTop: 0,
+    marginLeft:'15%',
```
Comment (aryanagarwal15) [inline swd/frontend/src/Routes/login/Loginbox.js]: use margin: 0 auto instead of marginleft

File: swd/frontend/src/Routes/login/Loginbox.js [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+    borderStyle: 'solid',
+    borderColor: '#074F57',
+    fontFamily: 'Montserrat',
+    color:'#074F57',
+    paddingLeft: 10,
+    borderWidth: 2,
+    fontSize: 17,
+    marginTop: 20,
+    marginLeft:'15%',
+    width: '70%',
```
Comment (aryanagarwal15) [inline swd/frontend/src/Routes/login/Loginbox.js]: width should increase

File: swd/frontend/src/Routes/login/Loginbox.js [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+    borderStyle: 'solid',
+    borderColor: '#074F57',
+    fontFamily: 'Montserrat',
+    color:'#074F57',
+    paddingLeft: 10,
+    borderWidth: 2,
+    fontSize: 17,
+    marginTop: 0,
+    marginLeft:'15%',
+    width: '70%',
```
Comment (aryanagarwal15) [inline swd/frontend/src/Routes/login/Loginbox.js]: width should increase

File: swd/frontend/src/Routes/login/Loginbox.js [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+  password: {
+    borderStyle: 'solid',
+    borderColor: '#074F57',
+    fontFamily: 'Montserrat',
+    color:'#074F57',
+    paddingLeft: 10,
+    borderWidth: 2,
+    fontSize: 17,
+    marginTop: 20,
+    marginLeft:'15%',
```
[the remark this answers is not in the document]
Comment (Pranav-Agarwal) [reply swd/frontend/src/Routes/login/Loginbox.js]: I don't want to center the element, I have given a particular margin as per the design

File: swd/frontend/src/Routes/login/Loginbox.js [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+import TextField from 'material-ui/TextField';
+import Snackbar from 'material-ui/Snackbar';
+import {withApollo} from 'react-apollo';
+import {withRouter} from 'react-router-dom';
+import Paper from 'material-ui/Paper';
+import {white, cyan400, cyan500} from 'material-ui/styles/colors';
+import backIcon from './back.svg';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
```
[the remark this answers is not in the document]
Comment (Pranav-Agarwal) [reply swd/frontend/src/Routes/login/Loginbox.js]: I am not sure how to go about this, as the style is a js object currently

File: swd/frontend/src/Routes/login/Loginbox.js [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+import Dialog from 'material-ui/Dialog';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+import Snackbar from 'material-ui/Snackbar';
+import {withApollo} from 'react-apollo';
+import {withRouter} from 'react-router-dom';
+import Paper from 'material-ui/Paper';
+import {white, cyan400, cyan500} from 'material-ui/styles/colors';
+import backIcon from './back.svg';
+import { withStyles } from '@material-ui/core/styles';
```
[the remark this answers is not in the document]
Comment (Pranav-Agarwal) [reply swd/frontend/src/Routes/login/Loginbox.js]: i will remove this import

File: swd/frontend/src/Routes/login/Login.js [hunk 7]
```diff
@@ -9,31 +9,32 @@ import {withRouter} from 'react-router-dom';
 import Paper from 'material-ui/Paper';
 import {white, cyan400, cyan500} from 'material-ui/styles/colors';
 import backIcon from './back.svg';
+import { withStyles } from '@material-ui/core/styles';
```
[the remark this answers is not in the document]
Comment (Pranav-Agarwal) [reply swd/frontend/src/Routes/login/Login.js]: i will remove this import

File: swd/frontend/src/Routes/login/Loginbox.js (hunk 2 above)
Comment (Pranav-Agarwal) [reply swd/frontend/src/Routes/login/Loginbox.js]: I think the box should be of fixed width

**Pull request closed without merging**